### DISCUSSION TITLE
Constant Not Synced

### DISCUSSION THREAD

**Discussion by filippovpa** [Chia-Network/chia-blockchain#3180] [General]
Since today its always Not Synced, and there is no connection to Full Node peer
I can manualy add connection, its go OK for about hour or so, and then again Not Synced, and connection just gone from the list

Windows 10, time sync - OK

**Comment by Cast39** [comment]
Have you tried adding a port forwarding in your router on port 8444 to your PC? Or maybe add a different introducer:

**Introducer**

- North Asia introducer-apne.chia.net:8444
- South Asia introducer-apse.chia.net:8444
- Western North America: introducer-or.chia.net:8444
- Eastern North America introducer-va.chia.net:8444
- Europe: introducer-eu.chia.net:8444

**Nodes**

- North America: node-or.chia.net:8444, 
- Europe: node-eu.chia.net:8444, 
- Asia: node-apne.chia.net:8444

**Reply by jacoblroberts** [reply]
For others with Xfinity, check my fix below.   Adding nodes will not help in the long term as these will eventually be considered a 'threat' and blocked by Xfinity.

**Comment by filippovpa** [comment]
> Have you tried adding a port forwarding in your router on port 8444 to your PC? Or maybe add a different introducer:
> 
> **Introducer**
> 
> * North Asia introducer-apne.chia.net:8444
> * South Asia introducer-apse.chia.net:8444
> * Western North America: introducer-or.chia.net:8444
> * Eastern North America introducer-va.chia.net:8444
> * Europe: introducer-eu.chia.net:8444
> 
> **Nodes**
> 
> * North America: node-or.chia.net:8444,
> * Europe: node-eu.chia.net:8444,
> * Asia: node-apne.chia.net:8444

Yeah i did 
It connect, go for sync, and then Not Synced again
Strange thing, that for 2 days it works perfectly fine, i got fully synced, and start Farming...

**Comment by Cast39** [comment]
> Yeah i did
> It connect, go for sync, and then Not Synced again
> Strange thing, that for 2 days it works perfectly fine, i got fully synced, and start Farming...

is the height in the Wallets tab as high as the peak height in the Full Node tab? If it does, here ends my wisdom, sorry

**Comment by rjulius23** [comment]
Ok my experience is that node.chia.net is super flaky. Sometimes i can connect to it forcefully, sometimes it just rolls and nothing happens. I think the issue is central, do not think it is a client issue.

**Comment by rjulius23** [comment]
Can we prioritize this issue ?

**Comment by filippovpa** [comment]
i cant even ping node-eu.chia.net
is it down ?

**Comment by rjulius23** [comment]
(venv) m@m-Super-Server:~/chia-plotting-scripts$ telnet node.chia.net 8444
Trying 54.71.94.197...

(venv) m@m-Super-Server:~/chia-plotting-scripts$ telnet node-apne.chia.net 8444
Trying 18.179.127.148...

(venv) m@m-Super-Server:~/chia-plotting-scripts$ telnet node-or.chia.net 8444
Trying 54.191.64.151...

(venv) m@m-Super-Server:~/chia-plotting-scripts$ telnet node-eu.chia.net 8444
Trying 18.130.174.242...

**Comment by rjulius23** [comment]
None of them works currently and the Farming status also changed to : **Not synced or not connected to peers**

**Comment by filippovpa** [comment]
> None of them works currently and the Farming status also changed to : **Not synced or not connected to peers**

thx, guess we just have to wait....

**Comment by rjulius23** [comment]
Has been down for 2 hours now... :(

**Reply by jack60612** [reply]
@rjulius23 have you tried using cloudflare dns? 1.1.1.1

**Reply by gromran** [reply]
> @rjulius23 have you tried using cloudflare dns? 1.1.1.1

Look at the date... 
most sync problems have been resolved since 1.1.6

**Comment by Wardus** [comment]
I have this same problem.  It shows me as connected, but my status is Not Synced.  HELP!

**Reply by jack60612** [reply]
try using cloudflare dns 1.1.1.1

**Comment by fiveangle** [comment]
Seeing same thing here.  All non-port mapped nodes fall out of sync regardless of the introducer.  Also, `chia show -a <local peer IP that has 8444 portmapped to the internet>:8444` results in that node refusing to stay connected.

Seems the P2P part of the Chia network is fundamentally not functioning consistently.

**Comment by filippovpa** [comment]
Still down?

**Comment by rjulius23** [comment]
Using **chiadb.net:8444** solved it for me, however the root cause is still unknown as **node.chia.net:8444** worked fine for 2 weeks.

**Comment by rjulius23** [comment]
Looking deeper into the issue, I think something with UPnP went fishy. I will play around with the UPnP settings in my router and in the chia node to see if that is the culprit.

**Comment by filippovpa** [comment]
> Using **chiadb.net:8444** solved it for me, however the root cause is still unknown as **node.chia.net:8444** worked fine for 2 weeks.

Finaly got sync in progress!! Thx a lot mate, ur my savior =)

**Comment by RollingRoland** [comment]
I have a similar problem. After I installed Chia 10 days ago things worked without any special router setup. But since 24 hours it is not syning or farming anymore. I tried everything in my shitty router to open the ports. It is a TP Link Archer C7. The closest I could find there was a port trigger option where tried to open the port. It does not seem to work. I also tried to manually add nodes and introducers in the GUI. After a few circles the window just disappeared and nothing happened. And on the 2 try on manually adding an introducer the window just hung itself and kept circling. After clicked the "restart" option the whole GUI is hung circling now. The Chia GUI is so full of bugs like I have never seen before. Unfortunately I am not an IT or Network specialist. So can't say I grasp on how to use CLI properly. I am really at the end of my wits. Any help would be much appreciated.

**Comment by filippovpa** [comment]
well....its Not Synced again...and chiadb.net not work anymore
why they call it 1.1.2 when its 0.0.1 at most...

**Comment by rjulius23** [comment]
I think the issue is with UPnP, do you have UPnP enabled on your router ? If yes i would suggest try to disable it and also run:

chia configure —upnp false

and restart the node.

**Comment by rjulius23** [comment]
If you disable UPnP port forwarding needs to be setup properly.

**Comment by filippovpa** [comment]
Can u help about  chia configure —upnp false ?

When i first run Chia, there was no Upnp, port 8444 was closed, but it works just fine
Then i got No Synced, trying to get Static IP, Upnp - ON, port OPEN, it works for some time, then it broke connection with all peers

Today i was able to connect to chiadb.net in like 5-6 attempts

How can u farm with this bugged coin if it keeps disconnecting with no reason...

**Reply by rjulius23** [reply]
I use no-ip to solve the static ip issue. If you open the port you dont need the UPnP. Unfortunately the local router can mess up UPnP. Another option is to restart the local router.

**Comment by kirk-clendinning** [comment]
On the subject of port 8444, which is the old plug and play, is no one worried about simply opening the router to their network and having issues with well known threats to UPnP? I've talked to people who simply enable 8444 on their home routers to enable their miners to run. However, my understanding is that opens an attack vector to IoT devices in one's home network. Is it possible to change the port to something less well known?

**Reply by fiveangle** [reply]
Yes, it's a sophomoric implementation. Ironically all the best Bittorrent clients learned decades ago that hard-coding the incoming port to 6881 was  a recipe for disaster in the face of NAT, and hence they all independently came up with auto baton-passing drop-box style methods for communicating auto-changing incoming ports on a per-instance basis and moved on... and today Bittorrent works great.  Does the fact that Chia has the exact same hard-coded incoming port number requirement as the original Bittorrent protocol mean that Bram has not learned anything about modern networking in the last 25 years ? 😢 

You can assign a different hard-coded incoming port in the client, yet the protocol still assumes port 8444 by default, making all non-8444 clients treated as second-class citizens on the Chianet, requiring explicit initiator-only communication methods that are not prioritized, which leaves those users reliant on the introducers. As we've seen, this is a critical weak link. Auto-changing and auto-assigned incoming ports needed to be the expected default for the entire network.  Not only to prevent every NATed network with more than a single Chia client from clobbering each other, but the hardcoded port requirement leaves the Chia network 100% vulnerable to the simplest port blocking technique for squelching Chianet traffic.  As a network engineer, it's disheartening to see this limitation in a "band new crypto for the future" 🤕 

I suspect they felt treating all connections as initiator-setup would add a level of inefficiency to it, but that is the price one has to pay for resiliency.  The overloaded introducers the last week+ has shown the fragility of of not doing so.  My hope is they do a hail-mary regroup/redesign at the same time they release the pooling plots changes else I worry about the future of Chianet.  All any country or ISP has to do is add the simplest port-blocking to 8444 and the entire network stops dead in it's tracks.

**Reply by Muffexx** [reply]
I have so far refrained from enabling UPnP on my router for this very reason. Also I have not opened port 8444. My node is still synced and I have "attempted proofs". I read that the only limitation without an open port is that I can have less connections to other peers (10 vs 50).
Is this correct? Because so far I have not received any XCH.

**Comment by kirk-clendinning** [comment]
Dave,

Thank you for your thoughtful response. I understand now.

Regards,

Kirk

**Comment by jacoblroberts** [comment]
This is still an issue.  I have a harvester connected to a full node.  8444 is open for syncing and 8447 is open for harvester.  Harvester connects fine.  Chia full node will go from synced to not synced.  I added some connections, pinged them on port 8444 and they are reachable.  This is a huge problem.  I know a ton about computers and am having trouble.  I feel for the 'average' farmer.   Syncing has been fine until transactions started.  I've stopped INFO logging as well to reduce the through put.  Nothing helps, this is hugely random and very annoying.

**Reply by jacoblroberts** [reply]
**FIXED
Xfinity...  you are the culprit!  Turns out that the new xfinity has 'xFi Advanced Security' turned on by default.  I noticed it was blocking 'threats'.  I turn off this setting and seems to be okay now.  For those in need, go to internet.xfinity.com, click on the 'more' tab and then click into 'My Services' and turn off advanced security. Hope this helps someone else.

PS - once you turn it off, it will take up to 10 minutes to propagate.

**Reply by fiveangle** [reply]
That's a red-herring... it may have made things worse for you than otherwise, but without the Xfinity Advanced Security feature enabled on the topmost Xfinity network tier service available, all nodes still periodically go out of sync.  Additionally, across 4 other friends (2 of which are network experts who work in in the CDN space), spanning 6 disparate ISPs, we have exhausted all possible network or client-side issues and have come to the conclusion that these sync issues are due to the implementation, not the individual networks.

**Reply by jacoblroberts** [reply]
So far so good for me.  I'll post back if it goes down.  I'm more siding with a network issue than code.  I downloaded a lot of torrents back in the day and never had sync issues other than internal firewall.  I doubt the man that invented the protocol would allow a lack luster implementation to go live.    Plus how is Xfinity going to justify trying to fix an issue after all these years of torrenting?  How often does someone get hacked outside of an internal issue? Either weak passwords, negligence or shady employees.   I appreciate the cloudflare type solution but they need to understand intent.  If I open ports, don't try to security it.  More granular settings perhaps?  For now it's off.

**Reply by jacoblroberts** [reply]
Just replying to say syncing has been perfect since my last post.  Even brought 2 more harvesters on line.  The only issue now is Windows updates rebooting my machine as it pleases.

**Reply by fiveangle** [reply]
Yes, it has been generally far improved for everyone since around Fri the14th, presumably because the influx of individual unique clients plateued.  But keep believing it was XFinity Advanced Security if it makes you feel warm and fuzzy 😄

**Reply by jacoblroberts** [reply]
Good thing I posted this on May 9th which makes me feel hella fuzzy.  Not sure what you are carrying on about, guy.  Do you work for Xfinity or something?  You sound very defensive.

**Comment by h3k** [comment]
Hey, did someone figured it out how to maintain stable connection? I have port 8444 open I manage to keep the node synced for 3-4 days and now again not synced I tried with node.chia.net:8444 and had a stable sync for 3-4 days again but now same story. What can I do to keep my sync active all the time...

**Reply by jacoblroberts** [reply]
I think everyone that posted here has resolved their issue.  I believe all are something internal causing the issue.  It could be as simple as your computer going to sleep and turning off network chatter to something blocking connections you are unaware of; firewall, ISP, poor switch or cabling losing packets.  It could even go as deep as failing ram or hard drive sector.   Of course I'm injecting some worst case scenarios but usually it is the network.  Have a read through this post, trouble shoot what you can and narrow down the possibilities.

**Reply by h3k** [reply]
I went through all the posts here but still not able to fix my issue. My PC is not going to sleep neither my network is turning off, my cables and switch is fine, hard drive and ram are brand new and not issues found. Regarding your point on firewall and ISP, why would I be able to connect to some peers for 3-4 days and after I am losing connection?

**Reply by jacoblroberts** [reply]
your ISP can start blocking requests that come in to frequently thinking they are attempting a ddos attack or something.  Xfinity does it.  Check my post above this one.

**Reply by fiveangle** [reply]
> I believe all are something internal causing the issue.

This was absolutely not the case for all instances.
